index.html

```diff
@@ -142,97 +157,94 @@ <h2 class="datatext">Profesional Art Printing Data</h2>
 				the industry's standard</p>
 
 				<ul class="btn">
-					<li class="btn-skills">html</li>
-					<li class="btn-skills">bootstrap</li>
-					<li class="btn-skills">Ruby</li>
+					<li class="btn-pro space">html</li>
+					<li class="btn-pro">bootstrap</li>
+					<li class="btn-pro">Ruby</li>
 				</ul>
 	
         <div class="btn-mid">
           <button class="btn-see-project">See Project</button>
         </div>
 		</div>
-	
-  <div class="about-me">
-    <h2>About me</h2>
-      
-    <p>Hello I’m a software developer! I can help you build a product,feature 
-      or website look through some of my work and experience!If you like what you see 
-      and have a project you need coded, don’t hestiate to contact me.
-    </p>
+  </section>
+
+      <div class="about-me about-desktop">
+      <div>
+        <h2>About me</h2>
+          
+        <p class="soft-dev">Hello I’m a software developer! I can help you build a product,feature 
+          or website look through some of my work and experience!If you like what you see 
+          and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </div>

```

Comment:
Reduce the space between the text and image on the left

Reply:
Hello, the picture of this area was not included. I dont understand as I only have an imafe on the right, not left. Could you offer more explanation as to what I am to do ?